### ISSUE TITLE
[BUG] Rollup properties are not working

### ISSUE THREAD

**Issue by Berdir** [mariosimao/notion-sdk-php#270]
**Describe the bug**

Rollup properties are not working

**To Reproduce**

Load a node from a database with a rollup property, save it again:

```
Uncaught PHP Exception Notion\Exceptions\ApiException: "body failed validation. Fix one:
body.properties.Project type.title should be defined, instead was `undefined`.
body.properties.Project type.rich_text should be defined, instead was `undefined`.
body.properties.Project type.number should be defined, instead was `undefined`.
body.properties.Project type.url should be defined, instead was `undefined`.
body.properties.Project type.select should be defined, instead was `undefined`.
body.properties.Project type.multi_select should be defined, instead was `undefined`.
body.properties.Project type.people should be defined, instead was `undefined`.
body.properties.Project type.email should be defined, instead was `undefined`.
body.properties.Project type.phone_number should be defined, instead was `undefined`.
body.properties.Project type.date should be defined, instead was `undefined`.
body.properties.Project type.checkbox should be defined, instead was `undefined`.
body.properties.Project type.relation should be defined, instead was `undefined`.
body.properties.Project type.files should be defined, instead was `undefined`.
body.properties.Project type.status should be defined, instead was `undefined`.
body.properties.Project type.type should be not present, instead was `"rollup"`.
body.properties.Project type.rollup should be not present, instead was `{"type":"array","array":[{"type":"select","select":{"id...`.
body.properties.Project type.name should be defined, instead was `undefined`.
body.properties.Project type.start should be defined, instead was `undefined`." at /app/vendor/mariosimao/notion-sdk-php/src/Exceptions/ApiException.php line 33
```

Note sure what that means exactly.

**Expected behavior**
No error.

**Code snippet**
If applicable, add code snippets to help explain your problem.

**Environment (please complete the following information):**
 - SDK Version latest

**Additional context**

**Comment by Berdir** [comment]
Workaround is to remove the property before sending:

```
    // Workaround for broken rollup property.
    $properties = $task_page->properties;
    unset($properties['Project type']);
    $task_page = $task_page->changeProperties($properties);
```

thinking about it, rollups are I guess read-only properties that you are not meant to change, so maybe you need to do something like that and remove them always from the sent properties?

**Comment by mariosimao** [comment]
Thanks for your feedback!

Fixed and released on v1.11.0
